include/capstone/x86.h

```diff
@@ -1899,6 +1899,10 @@ typedef enum  x86_insn_group {
 	X86_GRP_IRET,	// = CS_GRP_IRET
 	// all privileged instructions
 	X86_GRP_PRIVILEGE,	// = CS_GRP_PRIVILEGE
+	// relative and absolute branch groups
+	X86_GRP_BR,
+	// indirect branch groups
+	X86_GRP_IND_BR,
```

Comment:
for clarity, i think we should just use longer names. what do you think about X86_GRP_BRANCH_INDIRECT & X86_GRP_BRANCH_DIRECT?

same change can be applied for CS_GRP_BR & CS_GRP_IND_BR.